File: esports-backend/src/main/java/com/esports/dto/WalletResponseDTO.java
package com.esports.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class WalletResponseDTO {
    private Double availableBalance;
    private Double escrowBalance;
    private Double totalBalance;
}

File: esports-backend-dotnet/Upkart1/DTO/WalletResponseDTO.cs
namespace Upkart1.DTO
{
    public class WalletResponseDTO
    {
        public double availableBalance { get; set; }
        public double escrowBalance { get; set; }
        public double totalBalance { get; set; }

        public WalletResponseDTO() { }

        public WalletResponseDTO(double availableBalance, double escrowBalance, double totalBalance)
        {
            this.availableBalance = availableBalance;
            this.escrowBalance = escrowBalance;
            this.totalBalance = totalBalance;
        }
    }
}